test/fixtures/rules/std_rule_cases/AM08.yml

```diff
@@ -0,0 +1,140 @@
+rule: AM08
+
+test_pass_select_comma:
+  # this syntax is currently not covered by AM08
+  pass_str: "SELECT a.*, b.* FROM a, b"
+
+test_fail_missing_clause:
+  fail_str: "SELECT foo.a, bar.b FROM foo INNER JOIN bar;\n"
+  fix_str: "SELECT foo.a, bar.b FROM foo CROSS JOIN bar;\n"
+
+test_fail_missing_clause_lowercase_default:
+  fail_str: "SELECT foo.a, bar.b FROM foo left join bar;\n"
+  fix_str: "SELECT foo.a, bar.b FROM foo cross join bar;\n"
+
+test_fail_missing_multiple_joins:
+  fail_str: "SELECT foo.a, bar.b FROM foo left join bar on 1=2 left join baz;\n"
+  fix_str: "SELECT foo.a, bar.b FROM foo left join bar on 1=2 cross join baz;\n"
+
+test_fail_missing_multiple_joins_mixed_types:
+  fail_str: "SELECT foo.a, bar.b FROM foo left join bar on foo.x = bar.y left join baz;\n"
+  fix_str: "SELECT foo.a, bar.b FROM foo left join bar on foo.x = bar.y cross join baz;\n"
+
+test_fail_missing_multiple_joins_not_last:
+  fail_str: "SELECT foo.a, bar.b FROM foo left join bar left join baz on foo.x = bar.y;\n"
+  fix_str: "SELECT foo.a, bar.b FROM foo cross join bar left join baz on foo.x = bar.y;\n"
```

Comment:
I assumed that explicit cross join will scream error to code reviewer, but not fixing automatically also has clear merits.

I am comfortable with both approaches. I guess we just need majority consensus from maintainers.

Reply:
It's an interesting point. We have some existing rules which "correct" the code by making the implicit behavior _explicit_, which hopefully makes the user realize they made a mistake (e.g. AM01, AM02, AM03).

In this case, I think I agree with Michael that the user may not realize what a `CROSS JOIN` means, because they are rarely used in practice. For that reason, I think I prefer a lint error with no automatic "fix" here.

Reply:
Related question: In what cases do we think an automatic fix is appropriate (or not)? My initial intuition is that maybe we _never_ want an automatic fix for this.